cmd/xl-sets.go

```diff
@@ -1007,6 +1007,36 @@ func (s *xlSets) ReloadFormat(ctx context.Context, dryRun bool) (err error) {
 	return nil
 }
 
+// If it is a single node XL and all disks are root disks, it is most likely a dev setup, else it is a production setup.
+// On a dev setup we allow creation of format.json on root disks to help with testing.
+func isProductionSetup(infos []DiskInfo, errs []error) bool {
+	rootDiskCount := 0
```

Comment:
We can check if `globalIsDistXL` is true and then return true here.

Reply:
this is taken care of